src/txacme/test/test_challenges.py

```diff
@@ -172,4 +184,115 @@ def test_contains(self, underlay, overlay, key):
             Equals(proxy.get(key) is not None))
 
 
-__all__ = ['TLSResponderTests']
+class LibcloudResponderTests(_CommonResponderTests, TestCase):
+    """
+    `.LibcloudDNSResponder` implements a responder for dns-01 challenges using
+    libcloud on the backend.
+    """
+    _challenge_factory = challenges.DNS01
+    _challenge_type = u'dns-01'
+
+    def _responder_factory(self, zone_name=u'example.com'):
+        responder = LibcloudDNSResponder.create(
+            reactor=None,
+            driver_name='dummy',
+            username='ignored',
+            password='ignored',
+            zone_name=zone_name,
+            settle_delay=0.0)
+        responder._driver.create_zone(zone_name)
+        responder._ensure_thread_pool_started = lambda: None
+        responder._defer = execute
+        return responder
+
+    @example(token=b'BWYcfxzmOha7-7LoxziqPZIUr99BCz3BfbN9kzSFnrU',
```

Comment:
It's just a randomly generated value. I used the same one in all the examples (everything needs an `@example` because I effectively disable Hypothesis example generation in most of the test matrix configurations that run under coverage, since it's so slow) because I was lazy; pulling it out into a constant makes sense, though, so I'll do that.